tools/build-cross/Makefile-cross-aa64

```diff
@@ -0,0 +1,247 @@
+TARGET := pcsx-redux
```

Comment:
We really need to avoid this file, and have it be folded into the root's Makefile.